src/lib.rs

```diff
@@ -293,9 +292,8 @@ mod connection;
 mod cursor;
 mod statement;
 
-pub use connection::Connection;
-pub use connection::OpenFlags;
-pub use cursor::Cursor;
+pub use connection::{Connection, OpenFlags};
+pub use cursor::{ColumnIndex, Cursor, Row, ValueInto};
```

Comment:
Unfortunately yes because they are leaked in function arguments :sweat: 

...we could hide them from the doc using `#[doc(hidden)]` but imo it's not a great practice. The user usually like to know what's going on. Otherwise it feels a bit like dark magic.

But it's your call so let me know

Reply:
OK, makes sense.